pkg/loop/internal/relayer/pluginprovider/chainreader/helper_test.go

```diff
@@ -77,7 +77,7 @@ func (fakeTypeProvider) CreateContractType(_, itemType string, isEncode bool) (a
 			return &LatestParams{}, nil
 		}
 		return &TestStruct{}, nil
-	case MethodReturningUint64, DifferentMethodReturningUint64:
```

Comment:
Same comment here about removal. I was thinking this was a method with a different contract name. Maybe it's not necessary?

Reply:
answered in another comment